primitives/tf_encrypted/primitives/sodium/cc/easy_box_kernels.cc

```diff
@@ -159,15 +162,16 @@ class SodiumEasyBoxOpenDetached : public OpKernel {
 
     Tensor* plaintext_t;
     TensorShape plaintext_shape = ciphertext_t.shape();
+    plaintext_shape.RemoveLastDims(1);
```

Comment:
would be good to put in some assertions in the form of `OP_REQUIRES(context, ...)` here; for instance that the last dimension equals `sizeof(float)`